src/ORM/Connect/DBSchemaManager.php

```diff
@@ -640,8 +650,15 @@ public function requireField($table, $field, $spec)
             $specValue = $this->IdColumn(true);
         }
 
+        // Cache of fields
+        static $fieldListCache = [];
+
         if (!$newTable) {
-            $fieldList = $this->fieldList($table);
+            if (!isset($fieldListCache[$table])) {
+                $fieldListCache[$table] = $this->fieldList($table);
```

Comment:
Removed this, it completely breaks tests as we remove and re-build tables for each test we run... will need more thought if it’s to be implemented